### DISCUSSION TITLE
Change tffs on FB 7590

### DISCUSSION THREAD

**Discussion by sockd** [PeterPawn/YourFritz#64] [Q&A]
Hello,
I want to modify the tffs (change mac and password to make it "recovery-proof") of my 7590.
I created the tffs succesfully and uploaded it to the box with
./eva_store_tffs mtdnand NEW.img
But after booting and even after recovery the data is still the old.
Currently I´m running 7.57 on the box.
I hope you can help me.
Best regards.

**Comment by PeterPawn** [comment]
Some properties may not be changed permanently.

To disassemble the boot-loader configuration, you can use the `bootmanager` script (https://github.com/PeterPawn/YourFritz/blob/f0b0942846ffcc9ff8a5e8c038350fbd2f63e215/bootmanager/bootmanager#L75), you've to run it on the box itself. This script will show you, which settings can (or can't) be changed persistently.

Further persistent changes would need editing of this configuration area - not a really good idea, the gain is too small for the risk (even if this area exists two-times in a 7590 model).

**Reply by sockd** [reply]
Thanks for your reply.
If I run the script it shows me the values... thats fine, but the question is, how do i change the values "recovery-proof"...
In the past with 6490, 6591 and 7490 I could do this by modifying /proc/sys/urlader/environment (only for some variables) and mtdblock[n] (copy them away with dd, changed the values with sed and copied them back with dd).
But for the 7590 each change is reverted after a recovery...
So I´m asking for help to change for example the webgui_password and the maca address, so that they stay the new values even after a recovery...
